docs/DeveloperGuide.adoc

```diff
@@ -800,39 +802,57 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new student |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a student |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a student by name |locate details of students without having to go through the entire list
 
-|`* * *` |user |find a person by some characters |locate details of persons without having to remember his/her full name
+|`* * *` |user |find a student by some characters |locate details of students without having to remember his/her full name
 
-|`* * *` |user |filter a person by field |locate details of persons in a more optimized view
+|`* * *` |user |filter a student by field |locate details of students in a more optimized view
```

Comment:
Perhaps we want to consider adding field in glossary?

E.g. Field refers to the student's attributes. Some of the fields are mandatory using the `add` command and some of them will be updated later.